src/main/cljc/jarl/core.cljc

```diff
@@ -49,11 +51,10 @@
       (:help options)
       {:exit-message (usage summary) :ok? true}
       (not-empty arguments)
-      (let [ir (first arguments)
-            plan (second arguments)]
-        (if (jio/file? ir)
-          {:ir ir :plan plan :options options}
-          {:exit-message (error-msg [(fmt/sprintf "IR \"%s\" is not a file" ir)])}))
+      (let [[path entrypoint] arguments]
+        (if (jio/file? path)
+          {:path path :entrypoint entrypoint :options options}
+          {:exit-message (error-msg [(fmt/sprintf "IR \"%s\" is not a file" path)])}))
```

Comment:
Perhaps we should just drop the `IR ` prefix, as that's no longer part of the usage instructions.

Reply:
Agreed. Fixing.